subsys/bluetooth/mesh/Kconfig

```diff
@@ -289,6 +289,46 @@ config BT_MESH_CRPL
 	  protection list. This option is similar to the network message
 	  cache size, but has a different purpose.
 
+choice BT_MESH_RPL_STORAGE_MODE
+	prompt "Replay protection list storage mode"
+	default BT_MESH_RPL_STORAGE_MODE_SETTINGS
```

Comment:
I'm not sure if it is needed, but does it not make sence that it should use settings as a default? When adding the EMDS choice in DS it will be two options, and then it at least makes sence to me to use settings as a default, since the alternative would be to add the config option to all samples prj.conf. EMDS should IMO only be actively configured in by a user since it has HW requirements to operate correctly.

Reply:
Seems like other choice options does not have help text (ref choice BT_MESH_ADV). It might be sufficient that the different options has help text.